### ISSUE TITLE
can not start webapp

### ISSUE THREAD

**Issue by bazir123** [telegramdesktop/tdesktop#28022]
### Steps to reproduce

updated telegram to 5.1.4 x64 webapp wont open now
![screenshot 2024-06-07 03-35-47 001](https://github.com/telegramdesktop/tdesktop/assets/28486931/eae6511d-f0ab-42ef-bf06-4e881ae88aa6)
i am on windows 10 edge is default browser

### Expected behaviour

open webapp

### Actual behaviour

open webapp

### Operating system

windows 10

### Version of Telegram Desktop

5.1.4

### Installation source

Static binary from official website

### Logs

```shell
[2024.06.07 03:44:32] Launched version: 5001004, install beta: [TRUE], alpha: 0, debug mode: [TRUE]
[2024.06.07 03:44:32] Executable dir: C:/Soft/tel/, name: tel.exe
[2024.06.07 03:44:32] Initial working dir: C:/Soft/tel/
[2024.06.07 03:44:32] Working dir: C:/Soft/tel/
[2024.06.07 03:44:32] Command line: C:\Soft\tel\tel.exe
[2024.06.07 03:44:32] Executable path before check: C:/Soft/tel/tel.exe
[2024.06.07 03:44:32] Logs started
[2024.06.07 03:44:32] AppUserModelID: Telegram.TelegramDesktop.b29e23ce477c15b762e88def5d945a23
[2024.06.07 03:44:32] Using DirectX compiler 'C:\Soft\tel\modules\x64\d3d\d3dcompiler_47.dll'.
[2024.06.07 03:44:32] Connecting local socket to Global\72dd9ba197a2aa495b938a1912b0f0ad-{87A94AB0-E370-4cde-98D3-ACC110C5967D}...
[2024.06.07 03:44:32] This is the only instance of Telegram, starting server and app...
[2024.06.07 03:44:32] Moved logging from 'C:/Soft/tel/log_start0.txt' to 'C:/Soft/tel/log.txt'!
[2024.06.07 03:44:32] Global devicePixelRatio: 1.1875
[2024.06.07 03:44:32] QT_DPI_ADJUSTMENT_POLICY: AdjustDpi
[2024.06.07 03:44:32] Primary screen DPI: 96, Base: 96.
[2024.06.07 03:44:32] Computed screen scale: 100
[2024.06.07 03:44:32] DevicePixelRatio: 2
[2024.06.07 03:44:32] ScreenScale: 100
[2024.06.07 03:44:32] Media Info: Type Playback, default: {0.0.0.00000000}.{3c89eb0a-3f37-479b-9808-42f13ce62e28}, list: { "Digital Audio (HDMI) (High Definition Audio Device)" <{0.0.0.00000000}.{33d89df7-97b7-450f-9368-afa98db75ce8}>, "Speakers (Realtek High Definition Audio)" <{0.0.0.00000000}.{3c89eb0a-3f37-479b-9808-42f13ce62e28}>, "Speakers (High Definition Audio Device)" <{0.0.0.00000000}.{a13b1249-df97-485c-8756-2369923f69a6}>, "Internal AUX Jack (High Definition Audio Device)" <{0.0.0.00000000}.{a846e3ff-b791-4454-b885-5206e5a9ab7d}>, "Speakers (High Definition Audio Device)" <{0.0.0.00000000}.{c3dcb91e-f9c0-4e0a-8cdd-ecb564eea42f}> }, full list refresh: false
[2024.06.07 03:44:32] Media Error: Can't sync default device for type Capture, default: , list: { "Internal AUX Jack (High Definition Audio Device)" <{0.0.1.00000000}.{0d03f7ab-3d8c-4373-9b42-127084923caf}>, "Stereo Mix (Realtek High Definition Audio)" <{0.0.1.00000000}.{1a7cef31-fdb1-4b51-bda8-58047a56cb1f}>, "Line In (Realtek High Definition Audio)" <{0.0.1.00000000}.{2313007a-f68e-4bbc-b057-3cb4381aca27}>, "PC Beep (Realtek High Definition Audio)" <{0.0.1.00000000}.{53e26755-f0d6-4f70-a752-2573a5d21ad3}>, "Microphone (Realtek High Definition Audio)" <{0.0.1.00000000}.{82fc2fda-073b-4858-ae38-1a615f9d1662}>, "Internal AUX Jack (High Definition Audio Device)" <{0.0.1.00000000}.{a34316f0-a449-435b-b48a-abcf046b1a25}>, "Line In (High Definition Audio Device)" <{0.0.1.00000000}.{accda96c-690c-437a-9e61-1672632fcc2c}>, "Microphone (High Definition Audio Device)" <{0.0.1.00000000}.{bb3b467a-8c4a-426a-b88d-98f75e2f66a0}>, "Front Black In (Realtek High Definition Audio)" <{0.0.1.00000000}.{d8515502-5cef-4088-86d3-c4971a37dee3}>, "Rear Blue In (Realtek High Definition Audio)" <{0.0.1.00000000}.{e0b58e61-a839-4f7d-b1c0-6ed67a11a98b}> }
[2024.06.07 03:44:32] Media Error: Can't sync default device for type Camera, default: , list: {  }
[2024.06.07 03:44:32] App Info: reading settings...
[2024.06.07 03:44:32] App Info: reading encrypted settings...
[2024.06.07 03:44:32] Lang Info: Loaded cached, keys: 7016
[2024.06.07 03:44:32] Font: from ':/gui/fonts/OpenSans-Regular.ttf' loaded 'Open Sans'
[2024.06.07 03:44:32] Font: from ':/gui/fonts/OpenSans-Italic.ttf' loaded 'Open Sans'
[2024.06.07 03:44:32] Font: from ':/gui/fonts/OpenSans-SemiBold.ttf' loaded 'Open Sans'
[2024.06.07 03:44:32] Font: from ':/gui/fonts/OpenSans-SemiBoldItalic.ttf' loaded 'Open Sans'
[2024.06.07 03:44:32] Font: from ':/gui/fonts/Vazirmatn-UI-NL-Regular.ttf' loaded 'Vazirmatn UI NL'
[2024.06.07 03:44:32] Font: from ':/gui/fonts/Vazirmatn-UI-NL-SemiBold.ttf' loaded 'Vazirmatn UI NL'
[2024.06.07 03:44:32] OpenAL Logging Level: 3
[2024.06.07 03:44:32] Audio Error: No capture device found!
[2024.06.07 03:44:33] App Info: reading accounts info...
[2024.06.07 03:44:33] App Info: reading encrypted info...
[2024.06.07 03:44:33] App Info: reading map...
[2024.06.07 03:44:33] App Info: reading encrypted map...
[2024.06.07 03:44:33] App Info: reading encrypted user settings...
[2024.06.07 03:44:33] App Info: encrypted user settings read.
[2024.06.07 03:44:33] App Info: reading encrypted mtp data...
[2024.06.07 03:44:33] MTP Info: read keys, current: 5, to destroy: 0
[2024.06.07 03:44:33] Map read time: 46
[2024.06.07 03:44:33] App Info: reading encrypted mtp config...
[2024.06.07 03:44:33] App Info: reading map...
[2024.06.07 03:44:33] App Info: reading encrypted map...
[2024.06.07 03:44:33] App Info: reading encrypted user settings...
[2024.06.07 03:44:33] App Info: encrypted user settings read.
[2024.06.07 03:44:33] App Info: reading encrypted mtp data...
[2024.06.07 03:44:33] MTP Info: read keys, current: 5, to destroy: 0
[2024.06.07 03:44:33] Map read time: 6
[2024.06.07 03:44:33] App Info: reading encrypted mtp config...
[2024.06.07 03:44:33] Export Info: Destroy top bar by controller removal.
[2024.06.07 03:44:33] OpenGL: Using custom 'gpu_driver_bug_list.json'.
[2024.06.07 03:44:33] QWindowsIntegration::createPlatformOpenGLContext QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize -1, redBufferSize -1, greenBufferSize -1, blueBufferSize -1, alphaBufferSize 8, stencilBufferSize -1, samples -1, swapBehavior QSurfaceFormat::DefaultSwapBehavior, swapInterval 0, colorSpace QSurfaceFormat::DefaultColorSpace, profile  QSurfaceFormat::NoProfile)
[2024.06.07 03:44:33] GPU features: QSet("disable_program_cache")
[2024.06.07 03:44:33] Disabling program cache:  GpuDescription(vendorId=0x8086, deviceId=0x152, subSysId=0x3397103c, revision=9, driver: "igdumdim64.dll", version=10.18.10.5161, "Intel(R) HD Graphics""")
[2024.06.07 03:44:33] QWindowsOpenGLTester::supportedRenderers GpuDescription(vendorId=0x8086, deviceId=0x152, subSysId=0x3397103c, revision=9, driver: "igdumdim64.dll", version=10.18.10.5161, "Intel(R) HD Graphics""") 2 renderer:  QFlags(0x2|0x4|0x8|0x20|0x200)
[2024.06.07 03:44:33] Qt: Using EGL from libEGL
[2024.06.07 03:44:33] Qt: Using OpenGL ES 2.0 from libGLESv2
[2024.06.07 03:44:33] QWindowsEGLStaticContext::create Created EGL display 0x20fecc59e20 v 1 . 5
[2024.06.07 03:44:33] QWindowsIntegration::createPlatformOpenGLContext QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize -1, redBufferSize -1, greenBufferSize -1, blueBufferSize -1, alphaBufferSize 8, stencilBufferSize -1, samples 0, swapBehavior QSurfaceFormat::DefaultSwapBehavior, swapInterval 0, colorSpace QSurfaceFormat::DefaultColorSpace, profile  QSurfaceFormat::NoProfile)
[2024.06.07 03:44:33] OpenGL Profile: OpenGLES.
[2024.06.07 03:44:33] OpenGL Renderer: ANGLE (Intel, Intel(R) HD Graphics Direct3D11 vs_5_0 ps_5_0, D3D11-10.18.10.5161)
[2024.06.07 03:44:33] OpenGL Vendor: Google Inc. (Intel)
[2024.06.07 03:44:33] OpenGL Version: OpenGL ES 3.0.0 (ANGLE 2.1.10 git hash: e3f59e8d0c3e)
[2024.06.07 03:44:33] OpenGL Extensions: GL_NV_EGL_stream_consumer_external, GL_OES_texture_float_linear, GL_OES_element_index_uint, GL_EXT_draw_buffers, GL_NV_pack_subimage, GL_AMD_performance_monitor, GL_EXT_render_snorm, GL_ANGLE_instanced_arrays, GL_ANGLE_texture_compression_dxt5, GL_EXT_texture_rg, GL_EXT_instanced_arrays, GL_EXT_multisampled_render_to_texture, GL_OES_surfaceless_context, GL_CHROMIUM_copy_texture, GL_ANGLE_client_arrays, GL_EXT_map_buffer_range, GL_EXT_draw_buffers_indexed, GL_OES_compressed_ETC2_sRGB8_alpha8_texture, GL_OES_texture_stencil8, GL_OES_compressed_ETC2_sRGB8_texture, GL_OES_compressed_EAC_RG11_unsigned_texture, GL_EXT_texture_type_2_10_10_10_REV, GL_EXT_disjoint_timer_query, GL_EXT_debug_marker, GL_APPLE_clip_distance, GL_EXT_multi_draw_indirect, GL_OES_EGL_image_external_essl3, GL_ANGLE_polygon_mode, GL_EXT_texture_storage, GL_NV_framebuffer_blit, GL_EXT_texture_compression_s3tc_srgb, GL_EXT_texture_norm16, GL_OES_sample_variables, GL_OES_depth32, GL_OES_draw_buffers_indexed, GL_EXT_texture_filter_anisotropic, GL_EXT_draw_elements_base_vertex, GL_ANGLE_memory_size, GL_EXT_unpack_subimage, GL_OVR_multiview, GL_OES_compressed_EAC_R11_unsigned_texture, GL_OES_texture_half_float_linear, GL_ANGLE_depth_texture, GL_CHROMIUM_bind_uniform_location, GL_EXT_occlusion_query_boolean, GL_ANGLE_texture_compression_dxt3, GL_EXT_discard_framebuffer, GL_OES_packed_depth_stencil, GL_ANGLE_program_cache_control, GL_ANGLE_pack_reverse_row_order, GL_EXT_polygon_offset_clamp, GL_EXT_read_format_bgra, GL_CHROMIUM_lose_context, GL_EXT_robustness, GL_OES_texture_float, , GL_ANGLE_renderability_validation, GL_OES_mapbuffer, GL_EXT_color_buffer_half_float, GL_NV_fence, GL_EXT_color_buffer_float, GL_ANGLE_copy_texture_3d, GL_OES_get_program_binary, GL_EXT_texture_border_clamp, GL_OES_compressed_ETC2_punchthroughA_sRGB8_alpha_texture, GL_EXT_depth_clamp, GL_OES_texture_npot, GL_OVR_multiview2, GL_ANGLE_shader_pixel_local_storage, GL_ANGLE_provoking_vertex, GL_ANGLE_request_extension, GL_EXT_blend_func_extended, GL_OES_draw_elements_base_vertex, GL_OES_shader_multisample_interpolation, GL_EXT_debug_label, GL_ANGLE_robust_client_memory, GL_KHR_parallel_shader_compile, GL_OES_compressed_EAC_RG11_signed_texture, GL_OES_fbo_render_mipmap, GL_ANGLE_base_vertex_base_instance_shader_builtin, GL_WEBGL_video_texture, GL_EXT_texture_format_BGRA8888, GL_OES_EGL_image, GL_EXT_blend_minmax, GL_OES_compressed_ETC2_punchthroughA_RGBA8_texture, GL_ANGLE_base_vertex_base_instance, GL_ANGLE_clip_cull_distance, GL_OES_EGL_image_external, GL_CHROMIUM_bind_generates_resource, GL_ANGLE_multi_draw, GL_OES_texture_border_clamp, GL_ANGLE_texture_usage, GL_EXT_base_instance, GL_EXT_shader_texture_lod, GL_CHROMIUM_copy_compressed_texture, GL_ANGLE_texture_multisample, GL_CHROMIUM_sync_query, GL_ANGLE_framebuffer_multisample, GL_OES_compressed_EAC_R11_signed_texture, GL_OES_compressed_ETC2_RGB8_texture, GL_ANGLE_translated_shader_source, GL_KHR_debug, GL_OES_compressed_ETC2_RGBA8_texture, GL_OES_texture_half_float, GL_EXT_conservative_depth, GL_OES_depth24, GL_EXT_texture_compression_rgtc, GL_OES_rgb8_rgba8, GL_ANGLE_multiview_multisample, GL_EXT_clip_control, GL_EXT_texture_compression_dxt1, GL_OES_standard_derivatives, GL_NV_shader_noperspective_interpolation, GL_ANGLE_stencil_texturing, GL_EXT_frag_depth, GL_ANGLE_get_serialized_context_string, GL_EXT_EGL_image_external_wrap_modes, GL_EXT_clip_cull_distance, GL_NV_pixel_buffer_object, GL_EXT_sRGB, GL_ANGLE_framebuffer_blit, GL_EXT_texture_mirror_clamp_to_edge, GL_EXT_texture_compression_bptc, GL_ANGLE_get_tex_level_parameter, GL_OES_vertex_array_object, GL_EXT_float_blend, GL_ANGLE_lossy_etc_decode
[2024.06.07 03:44:33] EGL Extensions: EGL_EXT_create_context_robustness, EGL_ANGLE_d3d_share_handle_client_buffer, EGL_ANGLE_d3d_texture_client_buffer, EGL_ANGLE_surface_d3d_texture_2d_share_handle, EGL_ANGLE_query_surface_pointer, EGL_ANGLE_window_fixed_size, EGL_ANGLE_keyed_mutex, EGL_ANGLE_surface_orientation, EGL_ANGLE_direct_composition, EGL_ANGLE_windows_ui_composition, EGL_NV_post_sub_buffer, EGL_KHR_create_context, EGL_KHR_image, EGL_KHR_image_base, EGL_KHR_gl_texture_2D_image, EGL_KHR_gl_texture_cubemap_image, EGL_KHR_gl_renderbuffer_image, EGL_KHR_get_all_proc_addresses, EGL_KHR_stream, EGL_KHR_stream_consumer_gltexture, EGL_NV_stream_consumer_gltexture_yuv, EGL_ANGLE_stream_producer_d3d_texture, EGL_ANGLE_create_context_webgl_compatibility, EGL_CHROMIUM_create_context_bind_generates_resource, EGL_CHROMIUM_sync_control, EGL_EXT_pixel_format_float, EGL_KHR_surfaceless_context, EGL_ANGLE_display_texture_share_group, EGL_ANGLE_display_semaphore_share_group, EGL_ANGLE_create_context_client_arrays, EGL_ANGLE_program_cache_control, EGL_ANGLE_robust_resource_initialization, EGL_ANGLE_create_context_extensions_enabled, EGL_ANDROID_blob_cache, EGL_ANDROID_recordable, EGL_ANGLE_image_d3d11_texture, EGL_ANGLE_create_context_backwards_compatible, EGL_KHR_no_config_context, EGL_KHR_create_context_no_error, EGL_KHR_reusable_sync, 
[2024.06.07 03:44:33] OpenGL: QOpenGLContext created, version: 3.3.
[2024.06.07 03:44:33] OpenGL: [TRUE] (Window)
[2024.06.07 03:44:41] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:41] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:41] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:41] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:43] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:43] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:46] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:46] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:47] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:48] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:48] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:48] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:48] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:48] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:48] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:52] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:52] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:52] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:54] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:54] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:54] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:54] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:56] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:58] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:59] API Warning: not loaded minimal channel applied.
[2024.06.07 03:44:59] QTextCursor::setPosition: Position '8388607' out of range
[2024.06.07 03:44:59] QTextCursor::setPosition: Position '8388607' out of range
[2024.06.07 03:45:00] API Warning: not loaded minimal channel applied.
[2024.06.07 03:45:00] RPC Error: request 171 got fail with code 400, error USER_NOT_PARTICIPANT
[2024.06.07 03:45:00] RPC Error: request 170 got fail with code 400, error USER_NOT_PARTICIPANT
[2024.06.07 03:45:00] RPC Error: request 169 got fail with code 400, error USER_NOT_PARTICIPANT
[2024.06.07 03:45:00] RPC Error: request 159 got fail with code 400, error USER_NOT_PARTICIPANT
[2024.06.07 03:45:01] API Warning: not loaded minimal channel applied.
[2024.06.07 03:45:01] API Warning: not loaded minimal channel applied.
[2024.06.07 03:45:06] RPC Error: request 84 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2024.06.07 03:45:08] RPC Error: request 101 got fail with code 400, error WEBFILE_NOT_AVAILABLE
[2024.06.07 03:45:08] RPC Error: request 280 got fail with code 400, error WEBFILE_NOT_AVAILABLE
```

**Comment by nickyharpor** [comment]
same problem on Win 11

**Comment by Aokromes** [comment]
https://t.me/TelegramDesktopTalk/121351

**Comment by john-preston** [comment]
Ok, sorry guys, feel bad. Fixed in 5.1.5.
